src/core_plugins/tagcloud/public/tag_cloud_vis_params.html

```diff
@@ -11,6 +11,32 @@
     <div class="form-group">
       <label>Font Size</label>
       <div class="tag-cloud-fontsize-slider"></div>
+      <div class="tag-cloud-fontsize-inputs">
+        <div class="tag-cloud-fontsize-input">
```

Comment:
And then you can replace this class and the one on line 27 with `kuiBarSection`. Then you'll be able to delete these styles from `tag_cloud.less`.